.dotnet/src/Custom/VectorStores/VectorStoreClient.Convenience.cs

```diff
@@ -26,22 +26,6 @@ public virtual Task<ClientResult<VectorStore>> ModifyVectorStoreAsync(VectorStor
     public virtual ClientResult<VectorStore> ModifyVectorStore(VectorStore vectorStore, VectorStoreModificationOptions options)
         => ModifyVectorStore(vectorStore?.Id, options);
 
-    /// <summary>
-    /// Gets an up-to-date instance of an existing vector store.
-    /// </summary>
-    /// <param name="vectorStore"> The existing vector store instance to get an updated instance of. </param>
-    /// <returns> The refreshed vector store instance. </returns>
-    public virtual Task<ClientResult<VectorStore>> GetVectorStoreAsync(VectorStore vectorStore)
```

Comment:
Just for my context -- did we decide we didn't want these conveniences?

Reply:
the vector store apis are all marked as experimental so they will get another round of scrutiny and updates. For this, we already have this functionality in the lower layers so I figured there was no reason to duplicate it. happy to revert the removal though if you feel it is important